### ISSUE TITLE
Adding passing right on pre code

### ISSUE THREAD

**Issue by mzaini30** [forem/forem#9390]
It's in Dev:

![Screenshot_2020-07-19-22-45-16-96](https://user-images.githubusercontent.com/7939342/87877599-d54bf080-ca11-11ea-9b9b-efc287d4e3b1.png)

There's no padding right in pre code.

In my mind, padding right in pre code looks nice:

![Screenshot_2020-07-19-22-45-26-67](https://user-images.githubusercontent.com/7939342/87877641-f9a7cd00-ca11-11ea-9c79-1b714f82f235.png)

So, this is CSS code for padding right in pre code (CSS of my blog):

![Screenshot_2020-07-19-22-45-34-86](https://user-images.githubusercontent.com/7939342/87877659-1b08b900-ca12-11ea-8b70-6dc2eb552caa.png)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by Arun4033622** [comment]
Can I take this up?

**Comment by rhymes** [comment]
@Arun4033622 yeah! I'll assign you the issue

**Comment by mikeydorje** [comment]
Closing as stale. Assigned contributor opened PR #9514 in July 2020 but it was closed without merging. No activity since 2021.

If code block padding is still an issue, please open a new issue with current screenshots.
